modules/oadp-preparing-aws-credentials.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
+//
+// * rosa_backing_up_and_restoring_applications/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-preparing-aws-credentials_{context}"]
+= Preparing AWS credentials
+
+An AWS account must be ready to accept an OADP installation.
+
+.Procedure
+. Create the following environment variables by running the following commands:
++
+[NOTE]
+====
+Change the cluster name to match your ROSA cluster, and ensure you are logged into the cluster as an administrator. Ensure that all fields are outputted correctly before continuing.
+====
++
+[source,terminal]
+----
+$ export CLUSTER_NAME=my-cluster <1>
+export ROSA_CLUSTER_ID-$(rosa describe cluster -c ${CLUSTER_NAME} --output json | jq -r .id)
+export REGION=$(rosa describe cluster -c ${CLUSTER_NAME} --output json | jq -r .region.id)
+export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed 's|^https://||')
+export AWS_ACCOUNT_ID='aws sts get-caller-identity --query Account --output text'
+export CLUSTER_VERSION='rosa describe cluster -c ${CLUSTER_NAME} -o json | jq -r .version.raw_id | but -f -2 -d '.' '
+export ROLE_NAME="${CLUSTER_NAME}-openshift-oadp-aws-cloud-credentials"
+export SCRATCH="/tmp/${CLUSTER_NAME}/oadp"
+mkdir -p ${SCRATCH}
+echo "Cluster ID: ${ROSA_CLUSTER_ID}, Region: ${REGION}, OIDC Endpoint: 
+${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
++
+<1> Replace `my-cluster` with your ROSA cluster name.
+
+. On the AWS account, create an IAM policy to allow access to S3.
```

Comment:
Should/can this be a prerequisite to avoid shifting between the two systems mid-procedure?
Otherwise, you might want to mention that the substep is to be done in OCP.

Reply:
thanks, this wording was advised by QE as correct.